pandas/core/sparse/series.py

```diff
@@ -41,13 +41,12 @@
 # Wrapper function for Series arithmetic methods
 
 
-def _arith_method_SPARSE_SERIES(op, name, str_rep=None, default_axis=None):
```

Comment:
Considering moving this and _arith_method_SPARSE_ARRAY to ops, so that all of the funcs called by ops.add_special* are in ops.  (Also there is some duplication that may be removable).  Thoughts?